README.md

```diff
@@ -1,4 +1,33 @@
 # hivefriends
 
-A bunch of friends all around the world cherishing moments spent together. Upload and share albums of your events and meetings!
+A bunch of friends all around the world cherishing moments spent together.
+Upload and share albums of your events and meetings!
```

Comment:
I think we should mention here that this is a private and invite only project, at least for now.